include/usearch/index.hpp

```diff
@@ -621,8 +621,8 @@ class error_t {
 #else
     ~error_t() noexcept { raise(); }
     void raise() noexcept {
-        if (message_)
-            std::terminate();
+        // if (message_)
+        //     std::terminate();
```

Comment:
Let's not do this. THe point here is that error messages must be consumed and not thrown away, and the destructor checks that.
You probably needed this because at some point the error type was copied and only one of the copies was properly consumed.

Elsewhere, I fixed this by `std::move()`ing the result at return callsite. you can do the same

Reply:
Thank you for pointing that out!